src/Backoffice/Employee/Domain/Actions/StoreEmployeesAction.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\Domain\Actions;
+
+use Lightit\Backoffice\Employee\Domain\DataTransferObjects\EmployeeDto;
+use Lightit\Backoffice\Employee\Domain\Models\Employee;
+
+class StoreEmployeesAction
```

Comment:
```suggestion
class StoreEmployeeAction
```
It stores a single employee, not a collection of them. The file name should be updated as well.